keps/sig-node/2043-pod-resource-concrete-assigments/kep.yaml

```diff
@@ -27,7 +27,7 @@ stage: stable
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.21"
+# latest-milestone: "v1.21"
```

Comment:
@dashpole @vikaschoudhary16 @RenaudWasTaken @fromanirh @AlexeyPerevalov FYI, PRR approval will need to be part of the PR that sets this back to 1.21.